astropy/__init__.py

```diff
@@ -11,6 +11,55 @@
 
 from astropy.utils.system_info import system_info
 
+__all__ = [  # noqa: RUF100, RUF022
+    "__version__",
+    "__bibtex__",
+    # Subpackages (mostly lazy-loaded)
+    "config",
+    "constants",
+    "convolution",
+    "coordinates",
+    "cosmology",
+    "io",
+    "modeling",
+    "nddata",
+    "samp",
+    "stats",
+    "table",
+    "tests",
```

Comment:
The test I added in the second commit showed that `tests` was missing from my initial list - but we do actually document it, so that was clearly a mistake... Anyway, good to add tests!